Rb.Forms.Barcode.Pcl/BarcodeScanner.cs

```diff
@@ -246,6 +246,10 @@ public void OnCameraReleased()
 
         private static void OnBarcodeChanged(BindableObject bindable, Barcode oldValue, Barcode newBarcode)
         {
+            if (newBarcode == null) {
+                return;
+            }
```

Comment:
Is the case that null is being set to the barcode, when there is a barcode in the frame, then the frame changes and there's no longer a barcode in the frame?  If this is the case, then it could be possible that such a notification ("there's no longer a barcode in the frame!") could be helpful to clients of the API, so that a message could be shown or the user otherwise told that the camera's no longer looking at a barcode.  

If this is the case, and we agree that it could be helpful to a developer using this API to know when there's no longer a barcode in the frame, then the fact that null is being passed as an argument could be documented and handled by consumers of the client, or some other "magic value" could be used instead of null.  

For the BarcodeChanged bindable property, it could get passed a set of Empty Args to indicate absence of a barcode, but I think the BarcodeChangedCommand would have to be prepared to handle null.